ci/examples.sh

```diff
@@ -15,8 +15,8 @@ build_simple_make() {
     mkdir -p ${BUILD_DIR}
     make -C examples/simple -B \
         BUILD_DIR=${BUILD_DIR} \
-        CONFIG=${CONFIG} \
-        BOARD=${BOARD} \
+        MICROKIT_CONFIG=${CONFIG} \
+        MICROKIT_BOARD=${BOARD} \
```

Comment:
Thanks for changing this, `BOARD` is inconsistent with all the other projects we have so have been meaning to change it.

We will need to update the READMEs for each example though.